src/main/java/org/opensearch/ad/transport/PreviewAnomalyDetectorTransportAction.java

```diff
@@ -236,7 +244,7 @@ public void accept(GetResponse response) throws Exception {
                 if (!response.isExists()) {
                     listener
                         .onFailure(
-                            new OpenSearchException("Can't find anomaly detector with id:" + response.getId(), RestStatus.NOT_FOUND)
+                            new OpenSearchStatusException("Can't find anomaly detector with id:" + response.getId(), RestStatus.NOT_FOUND)
```

Comment:
Same as this comment https://github.com/opensearch-project/anomaly-detection/pull/150/files/713eb7970db4c021aed87d559f2eba68cb32e129#r680149988. we need to get concrete error message based on exception type. For `OpenSearchException`, we will return general error message rather than this concrete error message.